deploy/packer/base/scripts/packages.sh

```diff
@@ -41,8 +81,30 @@ sudo apt-get install -y \
     jq \
     rsyslog-gnutls \
     openvpn \
-    v4l2loopback-utils \
     lxc
 
 ## Clear mounts created in base image so fstab is empty in other builds...
 sudo sh -c 'echo "" > /etc/fstab'
+
+
+## Install v4l2loopback
```

Comment:
Moved out some of the setup that was done when building the app ami to here so it's clear what is being configured.